platform/podjs/src/browserPod.ts

```diff
@@ -8,6 +8,8 @@ import type {
     PolyIn,
     PolyOut,
     PolyNav,
+    Network,
```

Comment:
OK, not weird. This has changed now to EndPoint. You will need to merge from main.

Reply:
hehe yep this is the second failure which is valid. the first build failure was indeed weird but guess random thing as didn't happen again 😆